pkg/commands/inspect.go

```diff
@@ -31,6 +31,7 @@ func (opts *inspectOpts) Flags(fs *flag.FlagSet) {
 	fs.BoolVar(&opts.buildEnv, "build-env", false, "just print the build environment")
 	fs.BoolVar(&opts.buildEnvDesc, "describe-build-env", false, "describe the build environment")
 	fs.BoolVar(&opts.zipInfo, "zip-info", false, "just print the zip details")
+	fs.BoolVar(&opts.worktree, "worktree", false, "print worktree status (clean/dirty)")
```

Comment:
New flag lets you run `actions-go-build inspect --worktree` to see the worktree status.